expression/integration_test.go

```diff
@@ -10302,3 +10302,17 @@ func (s *testIntegrationSuite) TestIssue27610(c *C) {
 	tk.MustQuery("SELECT col1, COL2 FROM PK_TCOLLATION3966STROBJSTROBJ WHERE COL1 IN ('notexist','6') and col2 not in (\"abcd\");").
 		Check(testkit.Rows())
 }
+
+func (s *testIntegrationSuite) TestLastInsertId(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec(`use test;`)
+	tk.MustExec(`drop table if exists lastinsertid;`)
+	tk.MustExec(`create table lastinsertid (id int not null primary key auto_increment);`)
+	tk.MustQuery("SELECT @@last_insert_id;").Check(testkit.Rows("0"))
+	tk.MustExec(`INSERT INTO lastinsertid VALUES (NULL);`)
+	tk.MustQuery("SELECT @@last_insert_id, LAST_INSERT_ID()").Check(testkit.Rows("1 1"))
```

Comment:
`LAST_INSERT_ID` can also take an expression as an argument, https://dev.mysql.com/doc/refman/8.0/en/information-functions.html#function_last-insert-id
Will you support that?

Reply:
Sorry, I misunderstood the goal of this PR.